README.md

```diff
@@ -1,51 +1,64 @@
-# [Foundation for Sites](http://foundation.zurb.com)
+<p align="center">
+  <a href="http://foundation.zurb.com/">
+    <img src="https://user-images.githubusercontent.com/9939075/38782856-2a64a43e-40fa-11e8-89cd-e873af03b3c4.png" alt="Foundation for Sites 6" width="50%"/>
```

Comment:
Could need a min-width for the mobile GitHub site / design.

Reply:
Perfect :+1:

Reply:
Hmm actually inline styles are removed by GitHub. So I guess all we can do is setting width="50%" or width="384px". What do you prefer ?

Reply:
The px value should be better. Not sure but in one project I got it working with the owners (responsive styles).

Reply:
Can you remember what project it was ?

Reply:
It was https://github.com/airbnb/react-sketchapp/pull/249
https://github.com/airbnb/react-sketchapp/pull/249/files

Reply:
Interesting, the currently generated markup is `<img alt="react-sketchapp" src="https://camo.githubusercontent.com/7a75c94cd088b6ae393cc2648d6d7f9a45ff0b5a/68747470733a2f2f636c6475702e636f6d2f4d785356456b635f67622e706e67" data-canonical-src="https://cldup.com/MxSVEkc_gb.png" style="max-width:100%;">` so it seems like a bug to me on the side of GitHub which removes all inline rules if there are more than one.

Reply:
`style="max-width:100%;"` with the right natural maximum dimensions should still work.

Reply:
It works. Thanks.